### ISSUE TITLE
[Done} Apply Random Quantum Neural Network on Noisy Medical Image Recognition_QIAO

### ISSUE THREAD

**Issue by davidlearn** [XanaduAI/QHack2023#87]
### Project Name: 
Apply Random Quantum Neural Network on Noisy Medical Image Recognition
### Which challenges would you like to submit your project for?

- Hybrid Quantum-Classical Computing Challenge
- Quantum computing today!
- NVIDIA Challenge
- Amazon Braket Challenge

### Project Link:
https://github.com/YuNariai/project_QIAO

**Comment by davidlearn** [comment]
We should qualify for Quantum computing today!  and Hybrid Quantum-Classical Computing Challenge. Thanks.

**Comment by JordanAWS** [comment]
Hi there QIAO, it looks like you didn't use Amazon Braket in your project, is that correct?

**Comment by davidlearn** [comment]
No, we did not use Amazon Braket due to the short period of the open hackathon and will use it in the future.
________________________________
